app/v1/connectors/package.scala

```diff
@@ -23,5 +23,5 @@ package object connectors {
 
   type MtdIdLookupOutcome = Either[MtdError, String]
 
-  type DesOutcome[A] = Either[ResponseWrapper[DesError], ResponseWrapper[A]]
```

Comment:
This still says `DesError`

Reply:
Also, this type has been renamed to `DownstreamXxx` but everything else has been renamed to `IfsXxxx`